Update Actions & Trigger, Dye Colors, & add Sounds

TheRealPear opened the pull request:
### Actions & Triggers
* Changed descriptions on some table entries to clarify features some more
* Documented play-sound action (resolves #104)
* Fixed indentation to be 4-spaces instead of 2-spaces in example XML.

The only thing yet to be done is adding more examples to demonstrate each action. Pablo has already given me some resources to quickly put some together, however, this is the most time-consuming part for me.

### Sound Presets
Created a new page showing a list of sound presets that PGM ships with, allowing users to re-use it for a play-sound action. I've already linked to it from the new table in the Actions & Trigger page and it can be found under `Reference > Misc` from the navigation bar.

The only thing left in this area is to document all the possible sound key values in Minecraft 1.8, or at least provide a link where it'll be organized in a table similarly to PGM sound presets.

![image](https://github.com/PGMDev/Website/assets/20259871/a28743b4-0854-41f6-b4ab-39c887a001f3)


### Dye Colors
Added some description to when and where you can use dye colors in a standard map.xml file. I've also added a new column to the table to show the corresponding block damage value (which is also consistent across stained clay, stained glass block/panes, and wools) for each dye color when using the `team-color` attribute.

![image](https://github.com/PGMDev/Website/assets/20259871/2901d229-a339-48cc-b6ac-5822b6d9f803)

CoWinkKeyDinkInc commented on docs/reference/misc/sounds.mdx:
```diff
@@ -0,0 +1,160 @@
+---
+
+Sounds are used to draw a player's attention to certain events on a map. PGM comes with several preset sounds that can be quickly used for scenarios.
+It can be used in combination with other actions, as defined in [Action &amp; Triggers](/docs/modules/mechanics/actions-triggers#sound-attributes).
```
```suggestion
It can be used in combination with other actions, as defined in [Actions &amp; Triggers](/docs/modules/mechanics/actions-triggers#sound-attributes).
```

CoWinkKeyDinkInc commented on docs/reference/misc/sounds.mdx:
```diff
@@ -0,0 +1,160 @@
+      <tr>
+        <td>OBJECTIVE_MODE</td>
+        <td>
+          Used when <a href="/docs/modules/objectives/monument-modes">Monument Modes</a> is triggered.
```
```suggestion
          Used when <a href="/docs/modules/objectives/monument-modes">Monument Modes</a> are triggered.
```

CoWinkKeyDinkInc commented on docs/reference/misc/sounds.mdx:
```diff
@@ -0,0 +1,160 @@
+  </table>
+</div>
+
+Copied from: [Kyori Adventure docs - Sound](https://docs.advntr.dev/sound.html)
```
We should also add [`Sounds.json`](https://minecraft.wiki/w/Sounds.json#Sound_events) as a reference here. If Adventure/Bukkit has a more specific sound library, that can be used instead. Otherwise, a quick example on how to make a Key using `Sounds.json` from the `Sound File Used` column should be included.

```suggestion
Copied from: [Kyori Adventure docs - Sound](https://docs.advntr.dev/sound.html)
<br/>
List of Sound Keys can be found [here](https://minecraft.wiki/w/Sounds.json#Sound_events).
```

TheRealPear replied:
This would be a better link, albeit with some exceptions: https://minecraft.wiki/w/Sounds.json/Java_Edition_values_before_1.9

According to zzuf, `fireworks.largeBlast`, `fireworks.largeBlast_far`, and `item.fireCharge.use` cannot be used because of capitalization.

TheRealPear commented:
Added a link with a table of Sound Keys values and updated the Actions page to show it as well.

I think it would be a better idea for me (or someone else) to do a separate pull request for adding examples, though. It's an entirely different project to go through.

CoWinkKeyDinkInc commented:
Fair enough.
